flink-ml-core/src/test/java/org/apache/flink/ml/common/datastream/DataStreamUtilsTest.java

```diff
@@ -86,6 +87,29 @@ public void testAggregate() throws Exception {
         assertEquals("190", stringSum.get(0));
     }
 
+    @Test
+    public void testAggregateWithNonNeutralInitialAccumulator() throws Exception {
+        DataStream<Long> dataStream =
+                env.fromParallelCollection(new NumberSequenceIterator(0L, 19L), Types.LONG);
+        DataStream<String> result =
+                DataStreamUtils.aggregate(
+                        dataStream, new TestAggregateFuncWithNonNeutralInitialAccumulator());
+        List<String> stringSumList = IteratorUtils.toList(result.executeAndCollect());
+        assertEquals(1, stringSumList.size());
+        String stringSum1 = stringSumList.get(0);
+
+        env.setParallelism(env.getParallelism() + 1);
```

Comment:
Let's reset the parallelism as the default value at the end of this test.

Reply:
The parallelism of the environment would be reset in the `before()` method, which happens before each test case, so there is no need to reset the parallelism to the default value here.